tests/trace-viewer/trace-viewer.spec.ts

```diff
@@ -66,12 +81,18 @@ const test = playwrightTest.extend<{ showTraceViewer: (trace: string) => Promise
 
 let traceFile: string;
 
-test.beforeAll(async ({ browser }, workerInfo) => {
+test.beforeAll(async ({ browser, browserName }, workerInfo) => {
   const context = await browser.newContext();
   await context.tracing.start({ name: 'test', screenshots: true, snapshots: true });
   const page = await context.newPage();
   await page.goto('data:text/html,<html>Hello world</html>');
   await page.setContent('<button>Click</button>');
+  await page.evaluate(() => {
+    console.log('Info');
+    console.warn('Warning');
+    console.error('Error');
```

Comment:
```
console.log({ foo: { is: true, window: document.window }})
console.log([1, undefined, null, "foo", new Error("barfoo")])
console.log(() => {})
```

lets add a test for these too? So that they get correctly serialised and then deserialised/shown to the user.